javaparser-core/src/main/java/com/github/javaparser/ParseProblemException.java

```diff
@@ -0,0 +1,24 @@
+package com.github.javaparser;
+
+import java.util.List;
+
+/**
+ * Thrown when parsing problems occur during parsing with the static methods on JavaParser.
+ */
+public class ParseProblemException extends Exception {
```

Comment:
Yep I agree, checked exceptions are annoying :p

Le sam. 3 sept. 2016 à 17:21, Danny van Bruggen <notifications@github.com>
a écrit :

> In
> javaparser-core/src/main/java/com/github/javaparser/ParseProblemException.java
> <https://github.com/javaparser/javaparser/pull/447#discussion_r77438130>:
>
> > @@ -0,0 +1,24 @@
> > +package com.github.javaparser;
> > +
> > +import java.util.List;
> > +
> > +/**
> > + * Thrown when parsing problems occur during parsing with the static methods on JavaParser.
> > + */
> > +public class ParseProblemException extends Exception {
>
> Nah, I think I agree, it's the Java 8 world and we want to use the
> functional programming features, and checked exceptions are hated there.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/javaparser/javaparser/pull/447/files/c24b5256f3ebdb0903b48fc0c77106904658a525#r77438130>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AJu56OiXlQlQsxaWpQI5-6ByV-jskUIkks5qmZCKgaJpZM4JyGiu>
> .
>